pkg/workloads/console/controller.go

```diff
@@ -395,14 +395,39 @@ func buildJob(name types.NamespacedName, csl *workloadsv1alpha1.Console, templat
 	timeout := int64(csl.Spec.TimeoutSeconds)
 
 	username := strings.SplitN(csl.Spec.User, "@", 2)[0]
+	jobTemplate := template.Spec.Template.DeepCopy()
+
+	numContainers := len(jobTemplate.Spec.Containers)
+
+	// If there's no containers in the spec then the controller will be emitting
+	// warnings anyway, as the job will be rejected
+	if numContainers > 0 {
+		c := jobTemplate.Spec.Containers[0]
+
+		// Only replace the template command if one is specified
+		if len(csl.Spec.Command) > 0 {
+			c.Command = csl.Spec.Command
```

Comment:
I didn't create a test for the inverse of this condition, as it seemed overkill.
Maybe we can unit test this function in the future anyway, if we add more stuff to it.